.github/workflows/tests.yaml

```diff
@@ -36,6 +36,8 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python }}
+      - name: install external dependencies
```

Comment:
did you consider https://docs.github.com/en/actions/guides/about-service-containers over installing them,
i believe its worth a shot to try if you haven't

Reply:
Actually had not considered that. It looks quite handy so I'll definitely look into it. Thanks for the tip 👍🏻